pkg/minikube/cruntime/docker.go

```diff
@@ -106,8 +106,17 @@ func (r *Docker) SocketPath() string {
 
 // Available returns an error if it is not possible to use this runtime on a host
 func (r *Docker) Available() error {
-	_, err := exec.LookPath("docker")
-	return err
+	var err error
+	if _, err = exec.LookPath("docker"); err == nil {
```

Comment:
Won't this return if `docker` is found? But this doesn't guarentee that `cri-docker` is installed.

Reply:
Thanks, updated to return an error if docker is not found, then check for either `cri-dockerd` or `dockerd`.